flag_engine/identities/models.py

```diff
@@ -28,20 +47,23 @@ class IdentityModel:
     identity_features: typing.List[FeatureStateModel] = field(default_factory=list)
     identity_traits: typing.List[TraitModel] = field(default_factory=list)
 
+    def get_identity_traits_as_dict(self) -> typing.Dict[str, TraitModel]:
```

Comment:
I'm not sure this needs to be a separate method? But if we think it does, we should probably mark it as a 'private' method with a leading underscore? Unless I'm missing a usage for it outside of the class itself?